src/app/spans.rs

```diff
@@ -0,0 +1,106 @@
+use crate::trace_context;
+use linkerd2_error::Error;
+use opencensus_proto::trace::v1 as oc;
+use std::collections::HashMap;
+use std::{error, fmt};
+use tokio::sync::mpsc;
+
+const SPAN_KIND_SERVER: i32 = 1;
+const SPAN_KIND_CLIENT: i32 = 2;
+
+#[derive(Clone)]
+pub struct SpanConverter {
+    kind: i32,
```

Comment:
Nit: If this is one of two values it probably doesn't need to be an `i32`?

Reply:
...Unless that size is needed for making an `oc::Span`

Reply:
I'm guessing the wire protocol probably defines this as 32 bits.